scripts/installer

```diff
@@ -435,7 +435,8 @@ install() {
 # uninstall invokes kubectl delete with the built manifest.
 uninstall() {
   echo "Uninstalling ..."
-  kubectl delete -f $TMP_FILE
+  kubectl api-resources --verbs=list -o name | \
+    xargs -n 1 kubectl delete --ignore-not-found -l=app.kubernetes.io/instance=default,app.kubernetes.io/part-of=tekton-dashboard --all-namespaces
```

Comment:
is the `instance` label necessary here?

Reply:
I think it is. In case of multiple instances, instance should be unique even across namespaces. If it’s not the case it will uninstall all instances.

Reply:
I plan to add instance override in the installer.

Reply:
It seems like another thing that could be forgotten by a user, and doesn't appear to give any benefit in this scenario over specifying the namespace for example.

If we're relying on it being unique across namespaces we would need to check for this in the installer, but then what about users not using the installer? Maybe you've already considered this and I'm missing something...

Reply:
One namespace is not enough, resources are created into several namespaces.

True, would be nice to check for unicity at install time.

Reply:
Will take care of multiple instances in another PR.
Deployment is deleted first, then remaining resources.